### ISSUE TITLE
why status always undefined?

### ISSUE THREAD

**Issue by yuxinfeng** [HongQiQu/AppLinks#1]
I clone your project and replace the Domain? 
Then execute "adb shell dumpsys package d" but result always:
        Package: com.ihongqiqu.applinks
        Domains: client.map.baidu.com
        Status:  undefined

**Comment by jingle1267** [comment]
You also need create a `statements.json` file under you domains. Make sure http://yourdomains.com/.well-known/statements.json is exists and is right.

**Comment by yuxinfeng** [comment]
yes, i upload the file in my server, and use the https://digitalassetlinks.googleapis.com/v1/statements:list?source.web.site=https://client.map.baidu.com&relation=delegate_permission/common.handle_all_urls to check, all is  right.    but my app status still  undefined

**Comment by jingle1267** [comment]
Not just upload this file. Change the key in this file is also important.

**Comment by yuxinfeng** [comment]
I use your project. 
![image](https://cloud.githubusercontent.com/assets/4517041/20345242/382a7a44-ac31-11e6-900b-6c89a95665aa.png)
when i visit: https://ihongqiqu.com/.well-known/statements.json.  the sha256_cert_fingerprints different? and Status always undefined.

**Comment by jingle1267** [comment]
You can read [THIS](https://developer.android.com/training/app-links/index.html) and check you steps.

**Comment by chamnap** [comment]
@jingle1267, in that android documentation, I can't find anything related `statements.json`. They talked only `assetlinks.json`.

**Comment by jingle1267** [comment]
@chamnap `statements.json` is depressed.
